app/controllers/observations/downloads_controller.rb

```diff
@@ -4,6 +4,8 @@ module Observations
   class DownloadsController < ApplicationController
     before_action :login_required
 
+    MAX_DOWNLOADS = 120
```

Comment:
120 seems way too low.  Surely there's some compromise between half a million and 120?  It was originally designed to be able to render a million in a few seconds, so surely 1000 wouldn't be too many?  Has anyone actually tested the performance of this method?